packages/unuse-angular/CHANGELOG.md

```diff
@@ -1,5 +1,24 @@
 # unuse-angular
 
+## 1.1.0
```

Comment:
Looks like I am not allowed to use the breaking change ! syntax while we are in 0.x.x versions...

Reply:
Let's push another commit to remove `!`?

Reply:
I'm in bed on my phone 📱 🤣
I can't edit files right now with the GH App (/ I don't want to)

Will be up in 1-2 hours, but then also on the way to my office.

Or I can approve changes you are doing.

Reply:
OK, I think the issue here is that `unuse@1.0.0` has already been released, maybe let's try not `fixed` versions before v1.0?

Reply:
Sure, can try

Reply:
@Shinigami92 See #38